src/runtime/meter-toast-api.js

```diff
@@ -43,21 +44,31 @@ export class MeterToastApi {
     /** @private @const {!../components/dialog-manager.DialogManager} */
     this.dialogManager_ = deps.dialogManager();
 
+    const iframeArgs = this.activityPorts_.addDefaultArguments(
+      feArgs({
+        isClosable: true,
+        publicationId: deps.pageConfig().getPublicationId(),
```

Comment:
Default arguments populates some of these other values for you.

Reply:
Good catch!